docs/modules/ROOT/pages/admin.adoc

```diff
@@ -105,6 +105,37 @@ image::defender-admin-add-owner.png[Defender Admin Add Owner]
 [[timelocks]]
 == Timelocks
 
+=== Creating a Timelock Controller from Defender
+
+You can create and deploy a new Timelock Controller directly from Defender. To create a new Timelock, go to Admin and click on "Contracts" and then "Create timelock". You'll be taken to a simple form where you will be asked to provide the initial list of proposers and executors as well as the minimum delay for a proposal to be executed.
+
+In order to verify the contract on etherscan, you can find the source code and compiler settings below:
+
+The deployment uses a vanilla instance of the https://docs.openzeppelin.com/contracts/4.x/api/governance#TimelockController[TimelockController contract V4.X provided by the OpenZeppelin Contracts library].
+
+
+``` TimelockController.sol
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/governance/TimelockController.sol";
+
+```
```

Comment:
You don't really need to include this, just link to the contract source code in the repo on the exact version and we're good!

Reply:
Something like `https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.2.0/contracts/governance/TimelockController.sol` (assuming you took it from v4.2)

Reply:
Out of curiosity: for the bytecode used to deploy a TL, I created a fresh HardHat environment and deployed this exactly:

//SPDX-License-Identifier: Unlicense
pragma solidity ^0.8.4;

 import "@openzeppelin/contracts/governance/TimelockController.sol";

 ```

Is the bytecode going to differ in case of changes in filename, license text, whitespaces/tabs?

Reply:
That's an awesome question. The contract bytecode will be exactly the same _except_ for the [metadata hash](https://docs.soliditylang.org/en/v0.8.6/metadata.html?highlight=metadata#encoding-of-the-metadata-hash-in-the-bytecode). This is a hash of the contract's metadata (which includes the source code, among other stuff), so changing the source code changes the metadata hash.

However, source code verifiers (like etherscan) purposefully ignore the metadata hash during verification, so you can provide _any_ source code that compiles to the same bytecode.

Reply:
That's awesome thanks.